force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html

```diff
@@ -28,6 +28,27 @@
                 service-schedule-model={serviceScheduleModel}
             ></c-service-schedule-review>
         </template>
+        <c-modal header={labels.goBack} is-nested="true">
```

Comment:
this drops another background and looks very odd. is the is-nested not working correctly?

Reply:
let's make a WI to try to address this later. Not a dealbreaker now.